tests/unit/amo/components/TestAddon.js

```diff
@@ -393,22 +392,17 @@ describe('Addon', () => {
     expect(rootNode.querySelectorAll('.AddonDescription br').length).toBe(3);
   });
 
-  it('preserves certain HTML tags in the description', () => {
-    let description = '';
-    const allowedTags = [...allowedDescriptionTags];
-    // Ignore <br/> since it's checked elsewhere.
-    allowedTags.splice(allowedTags.indexOf('br'), 1);
-    // eslint-disable-next-line no-restricted-syntax
-    for (const tag of allowedTags) {
-      description = `${description} <${tag}>placeholder</${tag}>`;
-    }
-    const rootNode = renderAsDOMNode({ addon: { ...fakeAddon, description } });
-    // eslint-disable-next-line no-restricted-syntax
-    for (const tagToCheck of allowedTags) {
-      expect(
-        rootNode.querySelectorAll(`.AddonDescription-contents ${tagToCheck}`).length
-      ).toBe(1);
-    }
+  it('allows some HTML tags in the description', () => {
+    const root = shallowRender({
+      addon: {
+        ...fakeAddon,
+        description: '<b>super</b> <i>cool</i> <blink>add-on</blink>',
+      },
+    });
+    const contents = root.find('.AddonDescription-contents');
+    expect(contents.html()).toMatch(
```

Comment:
I think this could use `.toHaveHTML()` rather than `.html()).toMatch()`. I think you wouldn't need the RegExp then.

Reply:
It looks like [.toHaveHTML()](https://github.com/blainekasten/enzyme-matchers#tohavehtmlhtmlstring) matches the entire HTML so I'd have to include the enclosing div and all its classes.